index.css

```diff
@@ -51,6 +57,99 @@ body {
   }
 }
 
+footer > div > ul > li {
+  margin-left: 15px;
+}
+
+.about_lang > ul > li {
+  list-style-type: none;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
+  margin-top: 20px;
+}
+
+.headline > div > ul > li {
+  display: inline;
+}
+
+footer > div > ul {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  justify-content: center;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.works_det > div > ul {
+  margin-top: -10px;
+  margin-left: -25px;
+}
+
+#sideMenu {
+  display: none;
+  position: fixed;
+  top: 92px;
+  z-index: 3;
+  background-color: #3c3a39;
```

Comment:
- [ ] the menu should be visible only for the mobile version not for the desktop.

![mmm](https://user-images.githubusercontent.com/61562626/140933335-7a20205e-7cb5-49f4-8331-01f1c9ff4e33.gif)